arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/NonEmptyList.kt

```diff
@@ -552,3 +552,5 @@ public fun <A> Iterable<A>.toNonEmptyListOrNull(): NonEmptyList<A>? =
 
 public fun <A> Iterable<A>.toNonEmptyListOrNone(): Option<NonEmptyList<A>> =
   toNonEmptyListOrNull().toOption()
+
+public fun <A> A.toNonEmptyList(): NonEmptyList<A> = nonEmptyListOf(this)
```

Comment:
nonEmptyListOf is more readable 😁

Reply:
Yes, that is indeed our train of thought since Kotlin Std also doesn't have `1.toList()`.